agent/engine/docker_task_engine_test.go

```diff
@@ -730,7 +730,7 @@ func TestCreateContainerMergesLabels(t *testing.T) {
 		"com.amazonaws.ecs.task-definition-family":  "myFamily",
 		"com.amazonaws.ecs.task-definition-version": "1",
 		"com.amazonaws.ecs.cluster":                 "",
-		"key": "value",
+		"key":                                       "value",
```

Comment:
is this adjusted by gofmt?

Reply:
yes it is.  I have gofmt configured on every :wq (go-vim)